grype-server/pkg/scanner/scanner.go

```diff
@@ -98,11 +100,16 @@ func (s *Scanner) ScanSbomJson(sbom string) (*models.Document, error) {
 	if err != nil {
 		return nil, fmt.Errorf("unable to decode sbom: %v", err)
 	}
-	if formatOption == format.UnknownFormatOption {
-		return nil, fmt.Errorf("unknown SBOM format option: %v", formatOption)
+
+	// Validate returns error despite the syftSbom is valid so just log error and continue
```

Comment:
not sure I understand, is it an error or not?

Reply:
I got validation error: `unable to decode: EOF` from https://github.com/anchore/syft/blob/1cfa73732a4c2ce29117b55f0f7c0cff3133be3c/syft/formats/syftjson/validator.go#L22
The syftjson was created with syft. The validation shows this error. But the grype understand it. 
Maybe the validator needs to be fixed int syft.
https://mottaquikarim.github.io/dev/posts/is-json.decoder-broken-in-golang/

Reply:
so why we need this validation?

Reply:
It seems like something that was added by you, and I see that in the `Decode` function while calling to `IdentifyFormat(by)` the validation is being executed so we don't need to run it again.
https://github.com/anchore/syft/blob/1cfa73732a4c2ce29117b55f0f7c0cff3133be3c/syft/encode_decode.go#L29
https://github.com/anchore/syft/blob/1cfa73732a4c2ce29117b55f0f7c0cff3133be3c/syft/formats.go#L104